python/ray/workers/setup_runtime_env.py

```diff
@@ -101,3 +111,68 @@ def setup(input_args):
     command_separator = " && "
     command_str = command_separator.join(commands)
     os.execvp("bash", ["bash", "-c", command_str])
+
+
+def current_ray_pip_specifier() -> Optional[str]:
+    """The pip requirement specifier for the running version of Ray.
+
+    Returns a string which can be passed to `pip install` to install the
+    currently running Ray version.  Returns None if running on a version
+    built from source locally (likely if you are developing Ray).
+
+    Examples:
+        "1.4.0" if running the stable release
+        "https://s3-us-west-2.amazonaws.com/ray-wheels/master/[...].whl" if
+            running the nightly or a specific commit
+    """
+    if os.environ.get("RAY_CI_POST_WHEEL_TESTS"):
+        # Running in Buildkite CI after the wheel has been built.
+        return os.path.join("/ray/.whl", get_wheel_filename())
```

Comment:
Is it possible to avoid hardcoding this  as `/ray` and instead go to the relative path? The reason for this is that it is possible to test this locally (albeit not with the truly dev wheel) by downloading the wheel. We can get the relative path via something like:
```
os.path.join(Path(__file__).resolve().parents[3], ".whl", get_wheel_filename()
```

Reply:
Nice, good idea